motoman_gp25_support/package.xml

```diff
@@ -7,20 +7,23 @@
     <p>ROS-Industrial support for the Motoman GP25 (and variants).</p>
     <p>
       This package contains configuration data, 3D models and launch files
-      for Motoman GP25 and GP25-12 manipulators.
+      for Motoman GP25, GP25-12 and GP25-20 manipulators.
     </p>
     <p>
       <b>Specifications</b>
     </p>
     <ul>
-      <li>GP25 - Default</li>
-      <li>GP25-12 - Variant</li>
+      <li>GP25 - Default (YR-1-06VXH25-A0*)</li>
+      <li>GP25-12 - Variant (YR-1-06VXH25-A1*)</li>
+      <li>GP25-20 - Variant (YR-1-06VX25-20-A0*)</li>
     </ul>
     <p>
       Joint limits and maximum joint velocities are based on the information 
-      found in <em>YASKAWA MOTOMAN-GP25 INSTRUCTIONS (HW1485496)</em> version
-      <em>185194-1CD, rev 0</em> and in the <em>YASKAWA MOTOMAN-GP25-12 INSTRUCTIONS 
-      (HW1484890)</em> version <em>181846-1CD, rev 0</em>.
+      found in <em>YASKAWA MOTOMAN-GP25 INSTRUCTIONS (HW1484720)</em> manual
+      <em>180704-1CD, rev 4</em>, in the <em>YASKAWA MOTOMAN-GP25-12 INSTRUCTIONS 
+      (HW1484890)</em> manual <em>181846-1CD, rev 0</em>, and in the 
```

Comment:
just out of curiosity: why are the document nrs different? Were they incorrect previously?